scr/src/main/java/org/apache/felix/scr/impl/ComponentRegistry.java

```diff
@@ -732,7 +732,7 @@ public void updateChangeCount()
             final Timer timer;
             synchronized ( this.changeCountTimerLock ) {
                 if ( this.changeCountTimer == null ) {
-                    this.changeCountTimer = new Timer();
+                    this.changeCountTimer = new Timer("SCR Component Registry");
```

Comment:
Should we also make the `Timer` use a daemon thread here by passing true?

Reply:
I don't know about "should", but we obviously _could_ do that too.

Reply:
I've updated the `Timer` thread to be a daemon.

Reply:
Thanks, my thought is that I cannot think of any reason we would want this thread to hold up the JVM from exiting.  The Framework itself is responsible to make sure there is at least one non-daemon thread active while the framework is running.  But once the framework has stopped I think the JVM should be free to exit, even in the off chance that this timer is still active.